packages/webview_flutter/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.3.18+1
+
+* Remove the deprecated `author:` field from pubspec.yaml
```

Comment:
the version here doesn't match the pubspec.yaml

Reply:
yeah I just realized a version bump was missing on https://github.com/flutter/plugins/pull/2295